sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts

```diff
@@ -83,6 +84,18 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
       const decryptedText = uint8ArrayToString(decryptResult.result);
       assert.equal(text, decryptedText);
     });
+
+    it("the CryptographyClient can be created from a full KeyVaultKey object", async function() {
+      const keyName = testClient.formatName(`${keyPrefix}-${this!.test!.title}-${keySuffix}`);
+      const keyVaultKey = await client.createKey(keyName, "RSA");
+      const cryptoClient = new CryptographyClient(keyVaultKey, credential);
```

Comment:
This effectively shadows the `cryptoClient` variable set in the outer scope. Can we rename this one to `cryptoClientFromKey` to make this more clearer?